projects/player/src/app/rolling/rolling.page.ts

```diff
@@ -65,8 +62,8 @@ export class RollingPage implements AfterViewInit {
     private pathService: PathService,
   ) { }
 
-  ngAfterViewInit() {
-    console.log('ngAfterViewInit RollingPage');
+  ionViewWillEnter() {
```

Comment:
This is one of the keys to getting the Leaflet map to avoid the tiles only showing for the (0, 0) coordinate.